psutil/_psbsd.py

```diff
@@ -632,14 +632,8 @@ def name(self):
 
     @wrap_exceptions
     def exe(self):
-        if FREEBSD:
+        if FREEBSD or NETBSD:
             return cext.proc_exe(self.pid)
-        elif NETBSD:
-            if self.pid == 0:
-                # /proc/0 dir exists but /proc/0/exe doesn't
-                return ""
```

Comment:
Did you run tests (`make test`)? Looks like without this we'll get a failure. Possibly cwd() has the same issue.

Reply:
```
>>> psutil.Process(0)
psutil.Process(pid=0, name='system', started='14:22:25')
>>> psutil.Process(0).cwd()
'/'
>>> psutil.Process(0).exe()
Traceback (most recent call last):
  File "/tmp/venv/lib/python3.7/site-packages/psutil-5.6.3-py3.7-netbsd-8.99.43-amd64.egg/psutil/_psbsd.py", line 561, in wrapper
    return fun(self, *args, **kwargs)
  File "/tmp/venv/lib/python3.7/site-packages/psutil-5.6.3-py3.7-netbsd-8.99.43-amd64.egg/psutil/_psbsd.py", line 636, in exe
    return cext.proc_exe(self.pid)
FileNotFoundError: [Errno 2] No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/venv/lib/python3.7/site-packages/psutil-5.6.3-py3.7-netbsd-8.99.43-amd64.egg/psutil/__init__.py", line 775, in exe
    return guess_it(fallback=err)
  File "/tmp/venv/lib/python3.7/site-packages/psutil-5.6.3-py3.7-netbsd-8.99.43-amd64.egg/psutil/__init__.py", line 768, in guess_it
    raise fallback
  File "/tmp/venv/lib/python3.7/site-packages/psutil-5.6.3-py3.7-netbsd-8.99.43-amd64.egg/psutil/__init__.py", line 773, in exe
    exe = self._proc.exe()
  File "/tmp/venv/lib/python3.7/site-packages/psutil-5.6.3-py3.7-netbsd-8.99.43-amd64.egg/psutil/_psbsd.py", line 565, in wrapper
    raise AccessDenied(self.pid, self._name)
psutil.AccessDenied: psutil.AccessDenied (pid=0)
```

IMO it does not make sense to to treat or even list pid 0 as process as it is the kernel.